eclipse-scout-core/src/desktop/outline/OutlineAdapter.ts

```diff
@@ -272,7 +272,11 @@ export class OutlineAdapter extends TreeAdapter {
       };
 
       if (nodeModel.jsPageModel) {
-        jsPageModel = $.extend(true, {}, nodeModel.jsPageModel, jsPageModel);
+        // If the jsPageModel contains properties containing data objects deserialize them if possible
```

Comment:
A more concise version that is probably easier to read:
```suggestion
        // If the jsPageModel contains data objects, deserialize them if possible.
```